runtime/master/src/main/java/org/apache/nemo/runtime/master/resource/LambdaEvent.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.nemo.runtime.master.resource;
+
+import io.netty.buffer.ByteBuf;
+
+import java.io.Serializable;
+
+/**
+ * LambdaEvent contains various types for LambdaExecutor to deal with.
+ */
+public final class LambdaEvent implements Serializable {
+
+  /**
+   * Types supported in LambdaEvent.
+   */
+  public enum Type {
+    VM_RUN,
+    CLIENT_HANDSHAKE,
```

Comment:
what are these types? please add comments

Reply:
Those are stub types. Not currently used.

Reply:
then please remove unnecessary types or add comments